lib/matplotlib/backend_managers.py

```diff
@@ -56,14 +56,13 @@ class ToolManager(object):
         `LockDraw` object to know if the message is available to write
     """
 
-    def __init__(self, canvas):
+    def __init__(self, manager):
```

Comment:
but you can still do that, right, ``canvas.manager`` gives you the ``FigureManager``...